### ISSUE TITLE
Uncaught TypeError: Cannot set property 'className' of undefined

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/forces-and-motion-basics#225]
```
forces-and-motion-basics : fuzz : built : load
Uncaught TypeError: Cannot set property 'className' of undefined
TypeError: Cannot set property 'className' of undefined
    at new i (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1564:12760)
    at new e (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1572:6514)
    at e.s.name [as createView] (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1589:5693)
    at e.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1562:2516)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1562:25807)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1562:25872
Approximately 6/9/2017, 5:57:49 PM
forces-and-motion-basics : fuzz : built : run
Uncaught TypeError: Cannot set property 'className' of undefined
TypeError: Cannot set property 'className' of undefined
    at new i (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1564:12760)
    at new e (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1572:6514)
    at e.s.name [as createView] (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1589:5693)
    at e.initializeView (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1562:2516)
    at Array.<anonymous> (https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1562:25807)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1497052669736/forces-and-motion-basics/build/forces-and-motion-basics_en.html?postMessageOnLoad&postMessageOnError&postMessageOnBeforeUnload&fuzzMouse:1562:25872
Approximately 6/9/2017, 5:57:49 PM
```

Probably introduced from phetsims/scenery#624"

**Comment by jessegreenberg** [comment]
Fixed, closing.
